src/Nixfmt/Predoc.hs

```diff
@@ -130,6 +131,10 @@ instance (Pretty a, Pretty b) => Pretty (a, b) where
 instance (Pretty a, Pretty b, Pretty c) => Pretty (a, b, c) where
   pretty (a, b, c) = pretty a <> pretty b <> pretty c
 
+-- | Default indentation spacing.
+indentWidth :: Int
```

Comment:
Generally the indentation width configuration –just like the line length limit– should be done in a way that it is only used by the rendering engine. That way, it may simply be passed from the CLI as a parameter to `layout`.

A global constant, as defined here, would always require to change the code and recompile to configure the indentation depth.

Reply:
I understand this as a code refactoring for a first step towards making it configurable in the future. But now that I'm seeing that this is probably not entirely correct, it's probably best to make it configurable already, so that it can also be tested properly.

Reply:
And to make it configurable, we should probably use [editorconfig's `indent_size`](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#indent_size)!